blog.html

```diff
@@ -22,6 +22,119 @@
         <!--#include virtual="includes/_nav.htm" -->
         <div class="right">
             <h1 class="content-title">Blog</h1>
+            <article>
+                <h2 class="bullet">
+                    <a id="apache_kafka_370_release_announcement"></a>
+                    <a href="#apache_kafka_370_release_announcement">Apache Kafka 3.7.0 Release Announcement</a>
+                </h2>
+                TODO: January 2024 - Stanislav Kozlovski (<a href="https://twitter.com/0xeed">@BdKozlovski</a>)
+                <p>We are proud to announce the release of Apache Kafka 3.7.0. This release contains many new features and improvements. This blog post will highlight some of the more prominent features. For a full list of changes, be sure to check the <a href="https://downloads.apache.org/kafka/3.7.0/RELEASE_NOTES.html">release notes</a>.</p>
+                <p>See the <a href="https://kafka.apache.org/36/documentation.html#upgrade_3_7_0">Upgrading to 3.7.0 from any version 0.8.x through 3.6.x</a> section in the documentation for the list of notable changes and detailed upgrade steps.</p>
+                <p>
+                    In the last release, 3.6,
+                    <a href="https://kafka.apache.org/documentation/#kraft_zk_migration">the ability to migrate Kafka clusters from a ZooKeeper metadata system</a>
+                    to a KRaft metadata system was ready for usage in production environments with one caveat - JBOD was not yet available for KRaft clusters.
+                    This release, 3.7, we are shipping support for JBOD in KRaft. (See <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-858%3A+Handle+JBOD+broker+disk+failure+in+KRaft">KIP-858</a> for details)
```

Comment:
Hi @stanislavkozlovski, I will push an update this line to mark JBOD as early access in 3.7.0 blog release notes.

Reply:
https://github.com/apache/kafka-site/pull/579/files